prow/config.yaml

```diff
@@ -340,6 +340,7 @@ tide:
     kubernetes-sigs/cluster-api-provider-openstack: squash
     kubernetes-incubator/service-catalog: squash
   target_url: https://prow.k8s.io/tide.html
+  blocker_label: merge-blocker
```

Comment:
where must this label exist? who has privileges to apply it?

Reply:
currently though those with write access to the repo, it will be picked up on any issues on a repo under tide, and will block merges to that repo.

also more magically it will block to a specific branch if you put `blah branch: foo-branch` in the issue title. >.>